hathor/consensus/block_consensus.py

```diff
@@ -171,15 +172,15 @@ def update_voided_info(self, block: Block) -> None:
                     best_score = head_meta.score
                 else:
                     # All heads must have the same score.
-                    assert abs(best_score - head_meta.score) < 1e-10
-            assert isinstance(best_score, (int, float))
+                    assert best_score == head_meta.score
+            assert isinstance(best_score, int)
```

Comment:
Maybe set the type of `best_score: int | None = None` above, and then this could be `assert best_score is not None`

Reply:
Done.